src/Core/Services/GraphQLSchemaCreator.cs

```diff
@@ -251,6 +265,225 @@ private DocumentNode GenerateSqlGraphQLObjects(RuntimeEntities entities, Diction
             return new DocumentNode(nodes);
         }
 
+        /// <summary>
+        /// Helper method to traverse through all the relationships for all the entities exposed in the config.
```

Comment:
Add notes about how the directive gets added for different scenarios. 
1. FK constraint defined in the database. 
2. No FK constraint defined in the database, but relationship fields configured through the config file.